variables_test.go

```diff
@@ -1188,7 +1188,7 @@ func TestVariables_ListsAndNullability_DoesNotAllowListOfNonNullsToContainNull(t
 			{
 				Message: `Variable "$input" got invalid value ` +
 					`["A",null,"B"].` +
-					"\nIn element #1: Expected \"String!\", found null.",
+					"\nIn element #2: Expected \"String!\", found null.",
```

Comment:
question: do you know why was this saying element #1 was null before?

Reply:
no idea